src/client/components/pages/parts/admin-logs-table.js

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (C) 2023 Shivam Awasthi
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+import * as bootstrap from 'react-bootstrap';
+import * as utilsHelper from '../../../helpers/utils';
```

Comment:
You should consider importing only the components you need, instead of importing the entire package. This would otherwise lead to larger bundle sizes. 

Similarly [here](https://github.com/metabrainz/bookbrainz-site/pull/1001/files#diff-ec1b09b3858b2b94f7808ba9474ca2a6044e10b2222bae71c3be501cb1d41343R19) as well.